cms/static/js/views/pages/container.js

```diff
@@ -416,9 +418,25 @@ function($, _, Backbone, gettext, BasePage,
                         || (useNewVideoEditor === 'True' && blockType === 'video')
                         || (useNewProblemEditor === 'True' && blockType === 'problem')
                 ) {
-                    var destinationUrl = primaryHeader.attr('authoring_MFE_base_url')
-                        + '/' + blockType
-                        + '/' + encodeURI(primaryHeader.attr('data-usage-id'));
+                    var pathToNewXBlockEditor = `/${blockType}/${encodeURI(primaryHeader.attr('data-usage-id'))}`;
+                    var destinationUrl = `${primaryHeader.attr('authoring_MFE_base_url')}${pathToNewXBlockEditor}`;
+
+                    try {
+                        if (this.options.isIframeEmbed) {
+                            return window.parent.postMessage(
+                                {
+                                    type: 'newXBlockEditor',
+                                    message: 'Open the new XBlock editor',
+                                    payload: {
+                                        url: pathToNewXBlockEditor,
```

Comment:
Question: why are we sending a URL to the MFE? Wouldn't it be easier to send `blockType` and `usageId` as values in the payload, and let the MFE figure out the URL?

Reply:
[inform] Implemented in follow-up commit: 
https://github.com/openedx/edx-platform/pull/35694/commits/8d5b7e3bb77cf48352f1e7864636ac05b31810e1#diff-1b7855448b6086c2a313e0b9ecff179e991106aac52648d880f6f9d551e64577R434